cmd/esc/cli/env.go

```diff
@@ -352,3 +375,10 @@ func (cmd *envCommand) writePropertyEnvironmentDiagnostics(out io.Writer, diags
 
 	return nil
 }
+
+func (cmd *envCommand) printDeprecatedNameMessage(name string, ref environmentRef) {
+	msg := fmt.Sprintf(
+		"%sWarning: Referring to an environment name ('%s') without a project is deprecated.\nPlease use '%s/%s' or '%s' instead.%s",
```

Comment:
Should we have one of the tests pass in a non-project scoped environment and confirm we see this warning?